doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -366,38 +344,37 @@ Instructions
       .. figure:: images/click_import_board_info_button.png
          :align: center
 
-   #. Upload the board info you have generated from the ACRN config tool.
+   #. Upload the board XML you have generated from the ACRN board inspector.
 
-   #. After board info is uploaded, you will see the board name from the
+   #. After board XML is uploaded, you will see the board name from the
       Board info list. Select the board name to be configured.
 
       .. figure:: images/select_board_info.png
          :align: center
 
-#. Load or create the scenario setting by selecting among the following:
+#. Load or create the scenario configuration by selecting among the following:
 
    - Choose a scenario from the **Scenario Setting** menu that lists all
      user-defined scenarios for the board you selected in the previous step.
 
-   - Click the **Create a new scenario** from the **Scenario Setting**
-     menu to dynamically create a new scenario setting for the current board.
+   - Click the **Create a new scenario** from the **Scenario Setting** menu to
+     dynamically create a new scenario configuration for the current board.
 
-   - Click the **Load a default scenario** from the **Scenario Setting**
-     menu, and then select one default scenario setting to load a default
-     scenario setting for the current board.
+   - Click the **Load a default scenario** from the **Scenario Setting** menu,
+     and then select one default scenario configuration to load a predefined
+     scenario XML for the current board.
 
-   The default scenario configuration XMLs are located at
-   ``misc/vm_configs/xmls/config-xmls/[board]/``.
-   We can edit the scenario name when creating or loading a scenario. If the
-   current scenario name is duplicated with an existing scenario setting
-   name, rename the current scenario name or overwrite the existing one
-   after the confirmation message.
+   The default scenario XMLs are located at
+   ``misc/config_tools/data/[board]/``. You can edit the scenario name when
+   creating or loading a scenario. If the current scenario name is duplicated
+   with an existing scenario setting name, rename the current scenario name or
+   overwrite the existing one after the confirmation message.
```

Comment:
Where does the tool save this new scenario? Could it be overwriting the files we have in the repo if the user selects 'overwrite'?

Reply:
@szhen11 who can help clarify. Technically we have not made any change on the usage of the configuration editor in v2.4. The workflow may not be ideal now, and we do plan to refine it in v2.5.